python/ec2-cloudwatch/source.bat

```diff
@@ -0,0 +1,13 @@
+@echo off
```

Comment:
Lets remove this file. Generally I think it will confuse users more than help. Our examples assume that users have their tooling setup however they like while this assumes the use of virtualenv.